public/js/app.js

```diff
@@ -7,29 +7,32 @@ $(document).ready(() => {
       username: $("input#user-input").val().trim(),
       password: $("input#password-input").val()
     };
-    if ($('#isNewUser').prop('checked')){
+    if ($('#isNewUser').prop('checked')) {
       // user wants to create a new account
-      $.post("/api/signUp",userCredentials).then((res)=>{
+      $.post("/api/signUp", {
+        name: userCredentials.username,
+        password: userCredentials.password
+      }).then((res) => {
```

Comment:
$.post("/api/signUp",userCredentials).then(....  is a shorter and cleaner way of making post requests

Reply:
I was just using this temporarily to get data to tranfer into mySql we can remove this